dockerGrader/testServer.py

```diff
@@ -0,0 +1,121 @@
+import base64, boto3, botocore, logging, os, sys, json, subprocess, shutil
+from flask import Flask
+
+ACCESS_PATH = "projects/{project}/users/{googleId}/curr.json"
+CODE_DIR = "./submission"
+TEST_DIR = "/tmp/test"
+
+SUBMISSIONS = 'submissions'
+BUCKET = 'caraza-harter-cs301'
+session = boto3.Session(profile_name='cs301ta')
+s3 = session.client('s3')
+
+app = Flask(__name__)
```

Comment:
Is there an easy way to print the port number when we start?  This defaulted to 5000 for me (looks like it defaulted to 8080 for you given the docs).

Reply:
That's only because I use port forwarding for vagrant VM. 5000->8080 I think by default it will print out the port. I will check later.